packages/expo-build-properties/src/withBuildProperties.ts

```diff
@@ -3,14 +3,17 @@ import type { ConfigPlugin } from '@expo/config-plugins';
 
 import { withAndroidBuildProperties, withAndroidProguardRules } from './android';
 import { withIosBuildProperties, withIosDeploymentTarget } from './ios';
-import { validateConfig } from './pluginConfig';
+import { PluginConfigType, validateConfig } from './pluginConfig';
 
 const pkg = require('expo-build-properties/package.json');
 
 /**
  * Config plugin to customize native Android or iOS build properties for managed apps
+ * @param config ExpoConfig
+ * @param props `PluginConfig` from app.json or app.config.js
+ * @ignore
  */
-const withBuildProperties: ConfigPlugin<any> = (config, props) => {
+export const withBuildProperties: ConfigPlugin<PluginConfigType> = (config, props) => {
   const pluginConfig = validateConfig(props || {});
 
   config = withAndroidBuildProperties(config, pluginConfig);
```

Comment:
Just noticed that this plugin is memoized, I wonder if that's a good fit here. I imagine other plugins may want to utilize this plugin to bump certain versions. [ref](https://github.com/expo/expo/blob/880101488d9f6eb7788a506c121a0c516f86eb18/packages/expo-build-properties/src/withBuildProperties.ts#L27)

Reply:
makes sense. `createRunOncePlugin` removed. thanks!